src/YesSql.Provider.PostgreSql/PostgreSqlDialect.cs

```diff
@@ -93,6 +93,7 @@ public PostgreSqlDialect()
             Methods.Add("day", new TemplateFunction("extract(day from {0})"));
             Methods.Add("month", new TemplateFunction("extract(month from {0})"));
             Methods.Add("year", new TemplateFunction("extract(year from {0})"));
+            Methods.Add("now", new TemplateFunction("now() at time zone 'utc'"));
```

Comment:
```suggestion
            Methods.Add("now", new TemplateFunction("(now() at time zone 'utc')"));
```

Reply:
Using (now() AT TIME ZONE 'utc') would make this more bulletproof but I think it's not necessary. Just updated syntax since it should be upper case (just by convention).